scripts/pylib/twister/twisterlib.py

```diff
@@ -961,11 +961,6 @@ def handle(self):
         if harness.is_pytest:
             harness.pytest_run(self.log)
 
-        # sometimes a test instance hasn't been executed successfully with no
```

Comment:
I am wondering where else can we process those testcases with no status if the suite get passed, or just leave them as status null?

Reply:
Hi, @nashif , may I confirm this with you? Were you on purpose to make these testcases' status as "null", then trying to fix them in test case code later? Thanks!

Reply:
those cases will later be processed in `xunit_testcase` and get the right explanation which is a hint to fix the testsuite and make it compatible with twister.

Reply:
But we don't have similar process in `json_report`, they are still marked as `null` in twister.json, should we add this to `json_report`?

Reply:
Ok, thanks!